ZenPacks/zenoss/LinuxMonitor/FileSystem.py

```diff
@@ -155,7 +132,12 @@ def getStorageServer(self):
                     if ':' in storage_device:
                         search_keywords.add('has-nfs-client:{}'.format(storage_device))
 
-                search_root = self.getDmdRoot('Devices').Storage
+                # Check if we have a /Storage device class yet
+                try:
+                    search_root = self.getDmdRoot('Devices').Storage
+                except Exception:
+                    pass
```

Comment:
Passing here will result in search_root below raising an UnboundLocalError. You should probably return instead of pass.